system/fs/copy.go

```diff
@@ -1,27 +1,30 @@
 package fs
 
 import (
+	"fmt"
 	"io"
-	"log"
 	"os"
 )
 
 // Copy ...
 func Copy(source string, destination string) {
 	from, err := os.Open(source)
 	if err != nil {
-		log.Fatal(err)
+		fmt.Println("Unexpected error copying from source: ", err.Error())
+		os.Exit(1)
 	}
 	defer from.Close()
 
 	to, err := os.OpenFile(destination, os.O_RDWR|os.O_CREATE, 0666)
 	if err != nil {
-		log.Fatal(err)
+		fmt.Println("Unexpected error opening destination file handler: ", err.Error())
+		os.Exit(1)
 	}
 	defer to.Close()
 
 	_, err = io.Copy(to, from)
 	if err != nil {
-		log.Fatal(err)
+		fmt.Println("Unexpected error copying ["+source+"] to ["+destination+"]: ", err.Error())
+		os.Exit(1)
```

Comment:
I'm going to make a PR to change all instances of log.Fatal en masse, this is the better pattern for the mean time

Reply:
Let's discuss - I think don't bother. We can get all this stuff standardised in 2.0